internal/commands/login/inputs.go

```diff
@@ -27,11 +30,18 @@ type inputs struct {
 	PrivateAPIKey string
 	Username      string
 	Password      string
+	NoBrowser     bool
 }
 
 func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 	u := profile.Credentials()
 
+	if u == (user.Credentials{}) && !i.NoBrowser {
```

Comment:
the parentheses look weird, but I'll have learned something today if this works?

I think @andrewzlchen and I ran into this with metrics exporter previously and fell back to declaring:

```go
var (
  emptyThing typeThing
)
```

and then comparing equality against that: `if u == emptyThing { ... }`.  i've never seen this syntax before

Reply:
oo that's cool. I could try to drive-by this change in the next store PR!